cmd/verify_test.go

```diff
@@ -143,3 +149,82 @@ func TestHwAttestationPass(t *testing.T) {
 		})
 	}
 }
+
+func TestTdxAttestation(t *testing.T) {
+	file1, err := os.Create("attest")
+	if err != nil {
+		t.Fatal(err)
+	}
+	file2 := makeOutputFile(t, "verify")
+	defer os.RemoveAll(file1.Name())
+	defer os.RemoveAll(file2)
+
+	tpmNonce := "1234"
+	teeNonce := "6c62dec1b8191749a31dab490be532a35944dea47caef1f980863993d9899545eb7406a38d1eed313b987a467dacead6f0c87a6d766c66f6f29f8acb281f1113"
```

Comment:
Isn't this the same as TdxReportData? Why are we redefining it here?

Reply:
Yes, this should be a hex string.